style.css

```diff
@@ -0,0 +1,466 @@
+}
+
+#heading-description {
+  background-color: #f5f5f5;
+  box-shadow: rgb(0 0 0 / 20%) 0 0 6px;
+  padding: 10px;
+}
+
+.heading-contact-info {
+  padding: 10px;
+  font-size: 18px;
+}
+
+.text {
+  font-size: 20px;
+  font-weight: 900;
+  font-family: 'COCOGOOSE', Georgia, serif;
+}
+
+/* Statistics */
+#statistics {
+  background-image: url('./Assets/blackbackground.jpg');
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  color: #fff;
+}
+
+.statistics-grid {
+  display: flex;
+  gap: 10px;
+  flex-direction: column;
+}
+
+.stats-heading {
+  color: #ec5242;
+  margin-bottom: 30px;
+  border-bottom: 3px solid white;
+}
+
+.card {
+  background-color: #5f5f5f50;
+  display: flex;
+  padding: 20px 10px;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  gap: 20px;
+  transition: all 0.75s ease-in-out 0s;
+}
+
+.card::before {
+  border-bottom: 1px solid #ec5242;
+  border-left: 1px solid #ec5242;
+  transform-origin: 0 100% 0;
+}
+
+.card::after {
+  border-right: 1px solid #ec5242;
+  border-top: 1px solid #ec5242;
+  transform-origin: 100% 0 0;
+}
+
+.card:hover {
+  outline: 1px solid #ec5242;
+}
+
+.title {
+  color: #ec5242;
+}
+
+.description {
+  font-size: 14px;
+}
+
+#reg {
+  margin: 20px;
+  color: white;
+  padding: 20px;
+  background-color: #ec5242;
+  font-weight: bold;
+  border: none;
+  border-radius: 4px;
+  transition: all 0.5s;
+}
+
+#reg button:hover {
+  transform: translateY(-4px);
+}
+
+/* Teacher Section */
+#teachers-section {
+  background: white;
+  padding: 20px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.facilitators {
+  margin-bottom: 30px;
+  border-bottom: 3px solid #ec5242;
+}
+
+.facilitators-info {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  gap: 10px;
+  margin-top: 20px;
+}
+
+.facilitators-info__words {
+  display: flex;
+  flex-direction: column;
+}
+
+.facilitators-info hr {
+  width: 10%;
+  margin-top: 10px;
+}
+
+.teacher-img {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  width: 35%;
+}
+
+.teacher-img img {
+  width: 100%;
+  height: 100px;
+}
+
+.name {
+  display: flex;
+  align-items: start;
+  font-size: 1.5rem;
+}
+
+.name p {
+  margin-top: -10px;
+  font-weight: 700;
+}
+
+.designation {
+  font-style: italic;
+  font-weight: 400;
+  color: #ec5242;
+}
+
+.teacher-text {
+  font-size: 12px;
+}
+
+/* Sponsors Section */
+#sponsor-sections {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background-color: #272a31;
+  color: #8d8d8d;
+}
+
+#line {
+  width: 30px;
+  margin: auto;
+  border: 1px solid #ec5242;
+  background-color: #ec5242;
+}
+
+#sponsors-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 80%;
```

Comment:
**[OPTIONAL]**

- You can increase this with to 90% to look a little big to match the design.